### DISCUSSION TITLE
Receiving Mobile Version Release Information via Rocket.Chat Webhook

### DISCUSSION THREAD

**Discussion by Furret162** [RocketChat/Rocket.Chat.ReactNative#5930] [Q&A] [answered]
Is it possible to receive release information for the mobile version of Rocket.Chat through Rocket.Chat's webhook?

**Comment by diegolmello** [comment, marked as answer]
We're going to do it in the future, but it's a low priority.

I assume you'd like to have it similar to how new workspace versions work, right?
That banner is only to admins, so it might not be relevant enough to justify an implementation the way it is right now.
